src/hb-draw.hh

```diff
@@ -95,7 +100,35 @@ struct draw_helper_t
 	equal_to_current (to_x, to_y))
       return;
     if (!path_open) start_path ();
-    funcs->cubic_to (control1_x, control1_y, control2_x, control2_y, to_x, to_y, user_data);
+    if (funcs->is_quadratic_to_set)
+      funcs->cubic_to (control1_x, control1_y, control2_x, control2_y, to_x, to_y, user_data);
+    else
+    {
+      /* just an inaccurate one based https://github.com/raphlinus/font-rs/blob/master/src/raster.rs#L100 */
```

Comment:
@raphlinus Have used you quad-to-lines translation code, the formulas below aren't good for it I think, can you instruct me on how to improve and letting me know if I am ok to reuse the code here. Thanks